### ISSUE TITLE
Error streaming track

### ISSUE THREAD

**Issue by Asheboy** [mopidy/mopidy-soundcloud#101]
Hi,

I'm receiving the following error when attempting to play tracks:

```
INFO     'FSD Demo Mixdown' can't be streamed from SoundCloud
ERROR    SoundCloudBackend backend caused an exception.
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/mopidy/core/playback.py", line 423, in _change
    if not backend.playback.change_track(pending_tl_track.track).get():
  File "/usr/lib/python2.7/dist-packages/pykka/threading.py", line 52, in get
    compat.reraise(*self._data['exc_info'])
  File "/usr/lib/python2.7/dist-packages/pykka/compat.py", line 12, in reraise
    exec('raise tp, value, tb')
  File "/usr/lib/python2.7/dist-packages/pykka/actor.py", line 201, in _actor_loop
    response = self._handle_receive(message)
  File "/usr/lib/python2.7/dist-packages/pykka/actor.py", line 295, in _handle_receive
    return callee(*message['args'], **message['kwargs'])
  File "/usr/lib/python2.7/dist-packages/mopidy/backend.py", line 245, in change_track
    uri = self.translate_uri(track.uri)
  File "/usr/lib/python2.7/dist-packages/mopidy_soundcloud/actor.py", line 35, in translate_uri
    return track.uri
AttributeError: 'list' object has no attribute 'uri'
WARNING  Track is not playable: soundcloud:song/FSD Demo Mixdown.424869783
```

I'm using jaedb/Iris as a front-end and am running mopidy 2.1 & mopidy-soundcloud 2.0.2-3.

**Comment by kingosticks** [comment]
We released Release v2.1.0 20 days ago, can you please verify your issue still exists in that latest code.

**Comment by Asheboy** [comment]
Working on latest :+1:
